src/test/scala/edu/ie3/simona/model/participant/load/LoadModelScalingSpec.scala

```diff
@@ -0,0 +1,486 @@
+
+      }
+
+      "correctly account for the scaling factor when targeting at maximum power" taggedAs SnailTest in {
+        val scalingFactor = 1.5
+        val expectedMaximum = targetMaximumPower * scalingFactor
+        implicit val tolerance: Power = Watts(1d)
+        val dut = RandomLoadModel(
+          randomLoadInput.getUuid,
+          randomLoadInput.getId,
+          foreSeenOperationInterval,
+          scalingFactor,
+          QControl.apply(randomLoadInput.getqCharacteristics()),
+          Kilowatts(
+            randomLoadInput
+              .getsRated()
+              .to(PowerSystemUnits.KILOWATT)
+              .getValue
+              .doubleValue()
+          ),
+          randomLoadInput.getCosPhiRated,
+          ActivePower(targetMaximumPower)
+        )
+        dut.enable()
+        val powers = calculatePowerFromRelevantData(
+          simulationStartDate,
+          dut
+        ).sorted.toArray
+        /* Tolerance is equivalent to 10 W difference between the 95%-percentile of the obtained random results and the
+         * target maximum power. Because of the stochastic nature, the maximum power cannot be met perfectly */
+        RandomLoadModelSpec.get95Quantile(powers) =~ expectedMaximum
+      }
+    }
+  }
+
+  def getRelativeResult[Q <: squants.Quantity[Q]](
+      avgResult: Q,
+      expectedResult: Q
+  ): Dimensionless = {
+    val result = Percent(100) -
+      Percent(abs(avgResult.divide(expectedResult)))
+    Percent(abs(result.value.doubleValue()))
+  }
+
+  def getRelevantData[
+      C <: LoadRelevantData forSome { type LoadRelevantData },
+      T <: LoadModel[C] forSome { type C }
+  ](dut: T, simulationStartDate: ZonedDateTime) = {
+    dut match {
+      case _: RandomLoadModel =>
+        (0L until 35040)
+          .map(tick =>
+            tick -> RandomLoadModel
+              .RandomRelevantData(
+                simulationStartDate.plus(tick * 15, ChronoUnit.MINUTES)
+              )
+              .asInstanceOf[C]
+          )
+          .toMap
+      case _: ProfileLoadModel =>
+        (0L until 35040)
+          .map(tick =>
+            tick -> ProfileLoadModel
+              .ProfileRelevantData(
+                simulationStartDate.plus(tick * 15, ChronoUnit.MINUTES)
+              )
+              .asInstanceOf[C]
+          )
+          .toMap
+    }
+  }
+
+  def calculateAverageEnergy[
+      C <: LoadRelevantData forSome { type LoadRelevantData },
+      T <: LoadModel[C] forSome { type C }
+  ](
+      dut: T,
+      simulationStartDate: ZonedDateTime,
+      expectedEnergy: Energy
+  ): Dimensionless = {
+
+    val relevantDatas = getRelevantData(dut, simulationStartDate)
+
+    val totalRuns = 10
+    val avgEnergy = (0 until totalRuns)
+      .map { _ =>
+        relevantDatas
+          .map { case (tick, relevantData) =>
+            dut
+              .calculatePower(
+                tick,
+                Each(0d),
+                relevantData
+              )
```

Comment:
Someone has an idea for this? 
```
[Error] C:/Users/smdafeis/Documents/GitHub/simona/src/test/scala/edu/ie3/simona/model/participant/load/LoadModelScalingSpec.scala:438:16: ambiguous reference to overloaded definition,
both method calculatePower in trait ApparentPowerParticipant of type (tick: Long, voltage: squants.Dimensionless, data: C): edu.ie3.simona.agent.participant.data.Data.PrimaryData.ApparentPower
and  method calculatePower in class SystemParticipant of type (tick: Long, voltage: squants.Dimensionless, data: C): edu.ie3.simona.agent.participant.data.Data.PrimaryData.ApparentPower
match argument types (Long,squants.Dimensionless,Nothing)
```

Reply:
This took me way too long, but I have a solution now